### DISCUSSION TITLE
Reviving the iCubAnkara01: T1 – Motors causing S/D of power supply

### DISCUSSION THREAD

**Discussion by EbruBaglan** [robotology/community#664] [Q&A] [answered]
## Brief Description
- [ ] Q1. xantrex XFR 60-46 goes into S/D when the motors are turned on. How to spot the cause of the problem?
- [ ] Q2. Is there any button to start the head in this model (iCub 1.x)?

## Devices at Hand
-	iCubAnkara01 (iCub 1.x, version not exactly known)
-	Power Supply 1: xantrex XFR 35-35
-	Power Supply 2: xantrex XFR 60-46
-	iCub Laptop: DELL Latitude E4300

## Detailed Context

I have found an iCub in a research institution in Ankara, which is from around 20 years ago. Since I have done an internship at IIT, I thought of giving it a second life.

The model seems to be iCub 1.x as I learned from Marco Maggiali, which the documentation does not support anymore. You can find photos below to make sure. I even made a diagram of connections to spot the exact version of it, hopefully.

| <img src="https://github.com/robotology/community/assets/71343894/dc957a50-83e5-4004-897a-24b4884225ee" alt="drawing" width="150"/>  | <img src="https://github.com/robotology/community/assets/71343894/5defc9af-6299-482b-a2ae-f0a4c5280e39" alt="drawing" width="150"/>  | <img src="https://github.com/robotology/community/assets/71343894/98dc8e9d-6d35-4ffe-b899-8855905282f2" alt="drawing" width="450"/>  |
|---|---|---|
| iCubAnkara01  | The single output connection  |  The connection diagram |

I have two power supplies at hand. From a previous mailing with iCub Tech, I received the following voltage and amp values to set.
- Xantrex XFR 35-35: 12/12.5V-15A
- Xantrex XFR 60-46: 40V-20A

First, I set the limits as these and then I powered the iCub. I have a switch for turning on/off the CPU and motors as seen in connection diagram. I turned on CPU and no problem. But the moment **I turned on the motors switch, xantrex XFR 60-46 goes into S/D**.

So I guess there is a problem with the motors and I am looking for a solution to spot the cause.

Note: I have not pressed any button to start icub-head, is there any button in this model? According to the documentations, there is one in the newer models, but I do not know about this model.

Thank you so much in advance and hopefully this little guy will receive its second chance in life :)

**Comment by pattacini** [comment, marked as answer]
Hi @EbruBaglan 

Great that you aim to resurrect iCubAnkara01!

To let us follow up on this, it'd be better to post an issue on our [robot specific tickets system](https://github.com/robotology/icub-tech-support) as per these instructions: https://github.com/robotology/community?tab=readme-ov-file#-hw-support.

Transferring the exact same info therein would be super!

**Reply by EbruBaglan** [reply]
Hi @pattacini 

Thank you so much!😄 

Transferred the discussion to [this issue](https://github.com/robotology/icub-tech-support/issues/1671).

Let's resurrect the iFrankenCub!😄

**Reply by pattacini** [reply]
Thanks!
